src/main/java/org/opensearch/security/action/tenancy/MultiTenancyRetrieveAction.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.action.tenancy;
+
+import org.opensearch.action.ActionType;
+
+public class MultiTenancyRetrieveAction extends ActionType<BooleanSettingRetrieveResponse> {
+
+    public static final MultiTenancyRetrieveAction INSTANCE = new MultiTenancyRetrieveAction();
+    public static final String NAME = "securityconfig:admin/config/tenancy/multitenancy_enabled/read";
```

Comment:
I was thinking about making a new prefix instead of 'cluster'. What would you think about `feature:cluster/tenancy/multitenancy_enabled/(read|update)` instead?

I'd like it to be unassociated from where the setting is stored, it should be abstract enough it could move to another plugin, it should identicate the scope (e.g. cluster wide!), and then the local namespace tenancy followed by feature name.  

What do you think, other recommendations?

Reply:
I've updated it to `cluster:feature/tenancy/multitenancy_enabled/update` - but it seems like we should also update the rest request url to be inline with this, otherwise the coupling is still in place.